package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "snyk-protect": "snyk protect",
-    "test": "snyk test && node make test"
+    "test": "node make test"
```

Comment:
@jmervine: not sure if you noticed this; I had to add it because I don't like having to authenticate just for `npm test` to work. Also, makes me wonder how CI didn't error out...